src/Cameras/babylon.targetCamera.ts

```diff
@@ -287,6 +287,8 @@ module BABYLON {
                 // Computing target and final matrix
                 this.position.addToRef(this._transformedReferencePoint, this._currentTarget);
             } else {
+                //update the up vector!
+                BABYLON.Vector3.TransformNormalToRef(this._defaultUpVector, this._cameraRotationMatrix, this.upVector);

```

Comment:
why this line?
Is it necessary with the previous change?

Reply:
thanks for feedback.
It's mistake, I'll fix it.

Reply:
I fixed at this commit c9e4e5f